src/it/simple-it/pom.xml

```diff
@@ -101,6 +101,7 @@
                 <artifactId>@project.artifactId@</artifactId>
                 <version>@project.version@</version>
                 <configuration>
+                    <featureLauncherVersion>1.1.26</featureLauncherVersion>
```

Comment:
It would be good to add a comment saying why we pin the version here.

Reply:
sure.  I'll add a comment